install/traf_checkset_env.sh

```diff
@@ -0,0 +1,243 @@
+		exit 1
+	esac
+    fi
+    echo "INFO: command ${basecmd} exist"
+done
+
+osname=`uname -o`
+cpuarch=`uname -m`
+echo "INFO: os is [${osname}], cpu architecture is [${cpuarch}]"
+
+osdistributor=`lsb_release -i | cut -d : -f 2-`
+case ${osdistributor} in
+    *RedHat*|*CentOS*)
+	echo "INFO: os distributor id is [${osdistributor}]"
+	;;
+    *)
+	echo "ERROR: The OS distribution is [${osdistributor}], but Trafodion only supports RedHat and CentOS presently."
+	exit 1
+esac
+
+osver=`lsb_release -r | cut -d : -f 2-`
+osvermajor=`echo $osver | cut -d . -f 1-1`
+osverminor=`echo $osver | cut -d . -f 2-2`
+case ${osvermajor} in
+    6|7)
+	echo "INFO: os version is [$osver]"
+	;;
+    *)
+	echo "ERROR: OS version is [$osver]. Only 6.x versions are presently supported."
+	exit 1
+esac
+
+# install all libs
+dependlibs=(
+	    epel-release alsa-lib-devel ant ant-nodeps apr-devel	\
+	    apr-util-devel boost-devel cmake device-mapper-multipath	\
+	    dhcp doxygen flex gcc-c++ gd git glibc-devel		\
+	    glibc-devel.i686 graphviz-perl gzip ${MY_JAVA_VER}-devel	\
+	    libX11-devel libXau-devel libaio-devel libcurl-devel	\
+	    libibcm.i686 libibumad-devel libibumad-devel.i686 libiodbc	\
+	    libiodbc-devel librdmacm-devel librdmacm-devel.i686		\
+	    libxml2-devel lua-devel lzo-minilzo net-snmp-devel		\
+	    net-snmp-perl openldap-clients openldap-devel		\
+	    openldap-devel.i686 openmotif openssl-devel			\
+	    openssl-devel.i686 perl-Config-IniFiles perl-Config-Tiny	\
+	    perl-DBD-SQLite perl-Expect perl-IO-Tty perl-Math-Calc-Units \
+	    perl-Params-Validate perl-Parse-RecDescent perl-TermReadKey	\
+	    perl-Time-HiRes protobuf-compiler protobuf-devel		\
+	    readline-devel rpm-build saslwrapper sqlite-devel unixODBC	\
+	    unixODBC-devel uuid-perl wget xerces-c-devel xinetd		\
+	    ncurses-devel)
+faillibs=()
+echo "INFO: install dependent library start"
+for dependlib in ${dependlibs[@]}
+do
+    (${MY_YUM} install ${dependlib}) >>${LOGFILE}2>&1
+    if [ "x$?" != "x0" ]; then
+	faillibs=("${faillibs[@]}" "${dependlib}")
+	echo "ERROR: install dependent library [${dependlib}] fail"
+    else
+	echo "INFO: install dependent library [${dependlib}] success"
+    fi
+done
+echo "INFO: install dependent library finish"
+
+# remove pdsh and qt-dev
+echo "INFO: remove pdsh and qt-dev trafodion scripts get confused."
+(${MY_YUM} erase pdsh) >>${LOGFILE}2>&1
+(${MY_YUM} erase qt-dev) >>${LOGFILE}2>&1
+
+# check and set the java
+echo "INFO: check and set java environment"
+javadir=`\ls -L ${MY_JVM_PATH} | grep "${MY_JAVA_VER}-"`
+javapath="${MY_JVM_PATH}/${javadir}"
+dirs="ASSEMBLY_EXCEPTION  bin  include  jre  jre-abrt  lib  LICENSE" \
+				     " tapset  THIRD_PARTY_README"
+if [ ! -d ${javapath} ]; then
+     echo "ERROR: java dir [${javapath}] isn't right"
+     exit 1
+fi
+
+javahome=`grep JAVA_HOME ~/.bashrc | wc -l`
+if [ "x${javahome}" = "x0" ]; then
+    echo -en "export JAVA_HOME=${javapath}\n" >> $HOME/.bashrc
```

Comment:
When writing into the users .bashrc, I would recommend adding a comment that shows what tool wrote this. Something like:

```
# Lines added by traf_checkset_env.sh
exprt JAVA_HOME=...
```